src/swap/sideshift.js

```diff
@@ -185,7 +185,7 @@ const createFetchSwapQuote = (api: SideshiftApi, affiliateId: string) =>
     )
 
     if (fixedQuote.error) {
-      throw checkQuoteError(rate, request, fixedQuote.error.message)
+      await checkQuoteError(rate, request, fixedQuote.error.message)
```

Comment:
This fixes the reported issue but creates a handful of flow errors since flow isn't seeing that checkQuoteError always throws. To get around that limitation, and avoid ugly flow hacks, let's add the generic throw from line 92 right after this line